src/App.tsx

```diff
@@ -158,9 +159,13 @@ export function App() {
               />
               <Route
                 path={CheckoutRoutes.RIEPILOGO_PAGAMENTO}
```

Comment:
Why delete the guard on RIEPILOGO_PAGAMENTO?

Reply:
Maybe a mistake. I see that there was two guards:
![image](https://github.com/pagopa/pagopa-checkout-fe/assets/34749257/347fd495-0a0e-43b9-a2e0-44beadbd85a2)
I think only the one on the SessionItems.transaction needs to be restore, because we are not managing cvv data FE side anymore. Do you agree?

Reply:
yep :-)